SherpaFrontEnd.Tests/Acceptance/AnswerSurveyAcceptanceTest.cs

```diff
@@ -201,8 +201,8 @@ public async Task UserShouldSeeThankYouForReplyingWhenSuccessfulySubmittingForm(
                     Languages.ENGLISH, new[] { ResponseEnglish1, ResponseEnglish2, ResponseEnglish3 }
                 }
             }, Reverse,
-            HackmanComponent.INTERPERSONAL_PEER_COACHING,
-            HackmanSubcategory.DELIMITED, HackmanSubcomponent.SENSE_OF_URGENCY, Position);
```

Comment:
Did we mistake components and subcomponents in the past and they needed to be switched?

Reply:
yes, I found thay they were named incorrectly in the frontend DTOs